.github/workflows/build-and-test.yml

```diff
@@ -11,4 +13,12 @@ jobs:
     - uses: cachix/install-nix-action@v19
       with:
         nix_path: nixpkgs=channel:nixos-unstable
+        extra_nix_config: |
+          extra-access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
+          extra-experimental-features = nix-command flakes ca-derivations
+          extra-substituters = https://cache.nixos.org/ https://cache.garnix.io/
+          extra-trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g=
+          require-sigs = true
+          sandbox = true
+          extra-system-features = nixos-test benchmark big-parallel kvm
```

Comment:
Seems like none of these are necessary?

Reply:
These features come from https://github.com/cachix/install-nix-action#how-can-i-run-nixos-tests. I am going to remove them on this check. In theory, they aren't needed.